### PR TITLE
fix(move-copy-modal): User navigated inside folder on click

### PR DESCRIPTION
**Pull request by ChenCodes**
Changes in this PR:
- shouldNavigateOnItemClick passed in to QuickSearchItem
- renders Link when ```shouldNavigateOnItemClick``` is passed in and renders span when ```shouldNavigateOnItemClick``` is not passed in

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ChenCodes has signed the CLA. Thanks for the pull request!

File: src/features/quick-search/__tests__/QuickSearchItem-test.js [hunk 1]
```diff
@@ -75,12 +75,24 @@ describe('features/quick-search/QuickSearchItem', () => {
...
+    test('should render Link for item info when shouldNavigateOnItemClick is passed in', () => {
+        const wrapper = mount(<QuickSearchItem itemData={itemData} shouldNavigateOnItemClick />);
+        const itemName = wrapper.find('a.item-name');
+        expect(itemName.prop('href')).toEqual('/file/321');
+    });
+
+    test('should render span for item info when shouldNavigateOnItemClick is not passed in', () => {
+        const wrapper = mount(<QuickSearchItem itemData={itemData} />);
+        const itemName = wrapper.find('.item-name');
+        expect(itemName.prop('href')).toEqual(undefined);
```
Comment (Visylvius) [inline src/features/quick-search/__tests__/QuickSearchItem-test.js]: Would it be possible to add a snapshot in these cases to make sure that a link and span are getting rendered respectively?

**Pull request merged**